docs/source/tutorials/portfolio_optimization.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>'trained for 1000 times' -&gt; 'trained for 1000 iterations'</p><p> </p><p>Small thing, but earlier we talk about the 'cost' function, now we call it a 'loss'; maybe use the same term for both?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>